spec/features/a_user_can_subscribe_to_the_place_feed.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'A user subscribe to the place Atom feed' do
+  scenario 'and consult the complete correct feed page' do
+    CsvImportService.new('Place', Rails.root.join('spec/fixtures/csv/valid.csv'), state: 'active').save
+    CsvImportService.new('Place', Rails.root.join('spec/fixtures/csv/valid2.csv'), state: 'active').save
+
+    visit '/feed'
+
+    expect(page).to have_title(t('feed.title'))
+
+    page.assert_selector('entry author name', count: 2)
```

Comment:
Using factory girl, we could test directly the presence of our authors (but `author_name` is a private property we shouldn't give access to)